### ISSUE TITLE
Main Nav > edit spacing on words/icons

### ISSUE THREAD

**Issue by jpyo** [SCarrero/home#10]
![image](https://user-images.githubusercontent.com/8764348/71197448-a2fdc900-225f-11ea-85ef-3586ec5cbe9b.png)

match spacing of the elements in the nav bar to the attached screenshot

**Comment by SCarrero** [comment]
Can you send me SVGs for those shapes that are flush to the edge of the svg?  The ones I grabbed from older projects have gaps between the visible components and the viewbox container of the svg.

**Comment by jpyo** [comment]
added them inside this zip. let me know if these work

[menu-icons.zip](https://github.com/SCarrero/home/files/3985362/menu-icons.zip)

**Comment by SCarrero** [comment]
40px seems an awful lot on the right for small screen sizes.  I originally had the padding increasing at each breakpoint.  Was the 40px recommendation intended for large breakpoints?
![image](https://user-images.githubusercontent.com/20446484/71218333-15d26880-228f-11ea-835f-b36000e5b636.png)
Let me know if you want to back it off at the small breakpoints.

**Comment by jpyo** [comment]
yes, this is only for large breakpoints, sorry forgot to mention that @SCarrero

**Comment by SCarrero** [comment]
Please review.
